modules/tester/manifests/init.pp

```diff
@@ -6,15 +6,23 @@
 		ensure => directory,
 	}
 
+	if versioncmp( "$tester_config[php]", '7.0' ) >= 0 {
+		$phpunit_package_name = "phpunit.phar"
+		$phpunit_repo_url     = "https://phar.phpunit.de/phpunit.phar"
+	} else {
+		$phpunit_package_name = "phpunit-old.phar"
```

Comment:
We can just use `phpunit.phar` for both of these, should simplify the other changes.